content/slides/intro-to-jupyterlab.qmd

```diff
@@ -28,6 +33,43 @@ JupyterLab is not shared!
     * Your computations won't interfere with anyone else's
 :::
 
+## JupyterHub ![](/_media/jupyterhub_icon.png){width="5%" style="vertical-align:middle"}
+
+Log in to CryoCloud JupyterHub with your GitHub username
+
+* Important: select a 4GB server option
+* Your JupyterLab is dedicated, _not_ shared!
```

Comment:
This is no longer completely true. Compute resources are not isolated anymore, but you are guaranteed the minimum you select in the dropdown. E.g. if you select 4GB, you may be able to use more than 4GB of memory if the shared node has some free. Perhaps a speaker note on this? It's a complicated subject because we do want to communicate that your filesystem is dedicated and minimum compute resources are guaranteed.